onnxruntime/contrib_ops/cuda/bert/longformer_attention_impl.h

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 #include "core/providers/cuda/shared_inc/cuda_utils.h"
+#include <cublas_v2.h>
```

Comment:
>include <cublas_v2.h> [](http://example.com/codeflow?start=1&length=21)

This header gets included in both .cc and .cu file. Is it possible to clash with small #define there?

Reply:
Not sure what you mean.

I moved it here as this file uses cudaStream_t and cublasHandle_t which are defined via cublas_v2.h.

What #define potentially clashes?

Reply:
Ok